### PR TITLE
Gluay rails

### PR DESCRIPTION
**Pull request by Gluay88**

### CODE REVIEW

File: app/javascript/components/pages/BlogNew.js [hunk 1]
```diff
@@ -31,13 +31,21 @@ export class BlogNew extends Component {
   handleSubmit = () => {
     this.props.createBlog(this.state.newBlog);
     this.setState({ submitted: true });
+    debugger;
+    this.props.readBlog();
+    de;
```
Comment (iameddieyayaya) [inline app/javascript/components/pages/BlogNew.js]: ^ do you need the debugger here?

**Review by iameddieyayaya — COMMENTED** [review]
Overall looks good

File: app/javascript/components/pages/BlogNew.js [hunk 2]
```diff
@@ -31,13 +31,21 @@ export class BlogNew extends Component {
   handleSubmit = () => {
     this.props.createBlog(this.state.newBlog);
     this.setState({ submitted: true });
+    debugger;
```
Comment (VonTeacher) [inline app/javascript/components/pages/BlogNew.js]: Remove debug statements before pushing. :)

File: app/javascript/components/pages/BlogNew.js (hunk 1 above)
Comment (VonTeacher) [inline app/javascript/components/pages/BlogNew.js]: Looks like part of a leftover debugger. Please delete.

File: app/javascript/components/App.js [hunk 3]
```diff
@@ -64,21 +62,17 @@ export class App extends Component {
...
+        "Content-Type": "application/json",
       },
-      method: "DELETE"
-    })
-    .then(response => response.json()
-    .this(payload => this.readBlog())
-    
-    )
+      method: "DELETE",
+    }).then((response) => response.json().then((payload) => this.readBlog()));
```
Comment (VonTeacher) [inline app/javascript/components/App.js]: Let's keep method chains consistent. In other places, you start a new line for each `.then`, so consider doing that here.

File: app/javascript/components/pages/BlogEdit.js [hunk 4]
```diff
@@ -51,13 +51,11 @@ export class BlogEdit extends Component {
...
           </FormGroup>
-          <Button
-            name="submit"
-            onClick={this.handleSubmit}>
+          <Button name="submit" onClick={this.handleSubmit}>
             Edit Blog
           </Button>
         </Form>
-        {this.state.submitted && <Redirect to={`/catblog/${this.props.blog.id}`} />}
+        {this.state.submitted && <Redirect to={`/blogindex`} />}
```
Comment (VonTeacher) [inline app/javascript/components/pages/BlogEdit.js:58]: Does this need to be interpolated?

File: app/javascript/components/pages/BlogNew.js [hunk 5]
```diff
@@ -31,7 +31,6 @@ export class BlogNew extends Component {
   handleSubmit = () => {
     this.props.createBlog(this.state.newBlog);
     this.setState({ submitted: true });
-    debugger;
     this.props.readBlog();
     de;
```
Comment (iameddieyayaya) [inline app/javascript/components/pages/BlogNew.js]: What about this `de;`?

**Review by VonTeacher — CHANGES_REQUESTED** [review]
Left some comments on necessary cleanup and some questions to answer.

**Comment by VonTeacher** [comment]
Closing as this PR was identified as obsolete by @Gluay88

**Pull request closed without merging**